coins.json

```diff
@@ -1543,5 +1543,29 @@
       "clientPublic": "https://bsc-dataseed1.binance.org",
       "clientDocs": "https://eth.wiki/json-rpc/API"
     }
+  },
+  {
+    "id": "matic",
```

Comment:
let's use new name `polygon` and new symbol

Reply:
Our Coin name and symbol are still Matic and 'MATIC' respectively. It's only the company name that changed.

Reply:
`id` is the project name, not related to the coin itself. Polygon would be a proper name for it

Reply:
Thanks for clarifying. I'll change the id to "polygon". Should I have to regenerate the files again after changing the id? Also should I keep the name as Matic or change it to Polygon?

Reply:
Yes; and Is Matic a native token on side chain like BNB?

Reply:
Yes, Matic is the native token.

Reply:
from `https://polygon.technology/get-started/`, are you adding `Matic PoS` or `Matic Plasma`?

Reply:
Currently Matic Network is one single POS chain. For exit mechanism we use Plasma and POS bridges.